### PR TITLE
Fix doc latest ver Plugin 0.4.0, Scala 2.13.4, sbt 1.4.5

### PR DESCRIPTION
**Pull request by bjornregnell**

### CODE REVIEW

**Review by sjrd — APPROVED** [review]

File: docs/user/sbt.rst [hunk 1]
```diff
@@ -16,17 +16,17 @@ This generates the following files:
 
 * ``project/plugins.sbt`` to add a plugin dependency::
 
-    addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.4.0-M2")
+    addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.4.0")
 
 * ``project/build.properties`` to specify the sbt version::
 
-    sbt.version = 1.4.1
+    sbt.version = 1.4.5
```
Comment (sideeffffect) [inline docs/user/sbt.rst:23]: btw https://eed3si9n.com/sbt-1.4.6

Comment (bjornregnell) [reply docs/user/sbt.rst:23]: Should we really change this? 1.4.6 does not seem to be officially out. Maybe write "or later" somewhere?

Comment (sideeffffect) [reply docs/user/sbt.rst:23]: The link I posted above is about as official announcement as they come. I guess that with the current cadence of releasing sbt (which is good!), somebody didn't update the scala-sbt.org/download.html website. It's a manual process and easy to forget.

But this is really just a minuscule point :wink: if people use 1.4.1 or 1.4.5, Scala Steward will come and upgrade it anyway (and Mergify will merge it) to what will be at that time 1.4.7 or later :smile:

**Comment by LeeTibbert** [comment]
@bjornregnell  Boy are you fast!   

Please let me propose an alternate approach.   I have been waiting for
the scala-native.g8 PR to get community commentary, reviewed, and merged
before submitting a PR to bring scala-native doc proper into correspondence.

I propose that once again we use the software engineering principle of 
Single Point of Truth and drop all uses of explicit, hard-coded versions
of the software in the .g8 template.  We would use well chosen reference and
html links instead.  

  (roughly - needs wordsmithing:  The template provides the files listed below.  Software versions
  are current as of the last time the template was updated.  Since
  software versions change rapidly, one should check that 
  versions in the generated files have not been supersede (scala dex link here?)
  )

I believe the benefit of this approach is that `docs/user/sbt.rst` would not
need to change whenever the .g8 template. changes.  This eliminates
the chance of version skew. (Can anybody see a pattern in my recent
string of PRs.) and reduces both required expertise/memory-capacity
and maintenance effort.

If you concur with this approach, either you or I can submit a PR.
I am perfectly happy to stand down in favor of the engagement of
another community member.  Please advise.

File: docs/user/sbt.rst (hunk 1 above)
Comment (LeeTibbert) [reply docs/user/sbt.rst:23]: I find announcements of sbt versions are, to be kind, hard to find.
The [sbt repository](https://github.com/sbt/sbt) lists 1.4.6 as the latest
(now 27 days old) version.

**Comment by bjornregnell** [comment]
@LeeTibbert Good idea. Feel fre to go ahead an propose a solution to a single point of truth. I just with this PR threw in an easy fix as I had l short time off during a coffee break :). Also I'm unsure about the best practical solution to this, so perhaps better the maintainers decide where to put the canonical versions in this repo.

File: docs/user/sbt.rst (hunk 1 above)
Comment (bjornregnell) [reply docs/user/sbt.rst:23]: Yes, better say "or later" in the text as we might miss an upgrade.

Comment (ekrich) [reply docs/user/sbt.rst:23]: Scala Native itself uses `1.4.6`.

**Comment by LeeTibbert** [comment]
@bjornregnell  Sorry to be terse. I am on more of a "sip of water" break than a coffee/tea break.

Does PR #2129, which builds on this PR, address the concerns of this PR? 

If yes, your closing this PR as "superseded by PR #2129" or such might
reduce confusion for the folks doing the PR merge.

If no, what can we add to PR #2129,  or to the PR you mentioned as
following it?

Thanks,

Lee

**Comment by bjornregnell** [comment]
Closing this in favour of #2129, which takes a more general approach to versions by delegating to the .g8 template.

**Pull request closed without merging**